flow-launcher.tera

```diff
@@ -0,0 +1,201 @@
+    </Style>
+    <Style
+        x:Key="QuerySuggestionBoxStyle"
+        BasedOn="{StaticResource BaseQuerySuggestionBoxStyle}"
+        TargetType="{x:Type TextBox}">
+        <Setter Property="Background" Value="#{{ mantle.hex }}" />
+        <Setter Property="Foreground" Value="#{{ surface1.hex }}" />
+        <Setter Property="FontSize" Value="26" />
+        <Setter Property="Height" Value="42" />
+        <Setter Property="Padding" Value="0,0,0,0" />
+    </Style>
+    <Style
+        x:Key="WindowBorderStyle"
+        BasedOn="{StaticResource BaseWindowBorderStyle}"
+        TargetType="{x:Type Border}">
+        <Setter Property="BorderThickness" Value="2" />
+        <Setter Property="BorderBrush" Value="#{{ if(cond=flavor.dark, t=surface0.hex, f=surface1.hex) }}" />
+        <Setter Property="CornerRadius" Value="12" />
+        <Setter Property="Background" Value="#{{ mantle.hex }}" />
+    </Style>
+    <Style
+        x:Key="WindowStyle"
+        BasedOn="{StaticResource BaseWindowStyle}"
+        TargetType="{x:Type Window}">
+        <Setter Property="Width" Value="576" />
+        <Setter Property="RenderOptions.ClearTypeHint" Value="Enabled" />
+    </Style>
+
+    <Style
+        x:Key="PendingLineStyle"
+        BasedOn="{StaticResource BasePendingLineStyle}"
+        TargetType="{x:Type Line}">
+        <Setter Property="Stroke" Value="#{{ green.hex }}" />
+    </Style>
+
+    <!--  Item Style  -->
+    <Style
+        x:Key="ItemTitleStyle"
+        BasedOn="{StaticResource BaseItemTitleStyle}"
+        TargetType="{x:Type TextBlock}">
+        <Setter Property="Foreground" Value="#{{ if(cond=flavor.dark, t=subtext0.hex, f=text.hex) }}" />
+    </Style>
+    <Style
+        x:Key="ItemSubTitleStyle"
+        BasedOn="{StaticResource BaseItemSubTitleStyle}"
+        TargetType="{x:Type TextBlock}">
+        <Setter Property="Foreground" Value="#{{ if(cond=flavor.dark, t=overlay0.hex, f=text.hex) }}" />
+        <Setter Property="FontSize" Value="13" />
+    </Style>
+    <Style
+        x:Key="ItemNumberStyle"
+        BasedOn="{StaticResource BaseItemNumberStyle}"
+        TargetType="{x:Type TextBlock}">
+        <Setter Property="Foreground" Value="#{{ surface1.hex }}" />
+    </Style>
+    <Style
+        x:Key="ItemTitleSelectedStyle"
+        BasedOn="{StaticResource BaseItemTitleSelectedStyle}"
+        TargetType="{x:Type TextBlock}">
+        <Setter Property="Cursor" Value="Arrow" />
+        <Setter Property="Foreground" Value="#{{ text.hex }}" />
+    </Style>
+    <Style
+        x:Key="ItemSubTitleSelectedStyle"
+        BasedOn="{StaticResource BaseItemSubTitleSelectedStyle}"
+        TargetType="{x:Type TextBlock}">
+        <Setter Property="Cursor" Value="Arrow" />
+        <Setter Property="Foreground" Value="#{{ blue.hex }}" />
+    </Style>
+    <SolidColorBrush x:Key="ItemSelectedBackgroundColor">#{{ if(cond=flavor.dark, t=base.hex, f=surface0.hex) }}</SolidColorBrush>
```

Comment:
> If the contrast is that bad on latte is it an issue with our WCAG contrast compliance with latte in general or should we change the color of that for not just latte?

Regarding the selection highlight, the reason for the diff is because you can get away with making the selection highlight `base` on the darker flavours but not on Latte. Since then I've added the following into the style guide for the selection highlight, would be good to change the colour to the one below so that you don't have to do an if condition:

![image](https://github.com/user-attachments/assets/42b299a5-36d9-4a6e-adc7-bed174f56090)